stdlib/Cargo.toml

```diff
@@ -32,6 +32,7 @@ blake3 = "1.5"
 miden-air = { package = "miden-air", path = "../air", version = "0.8", default-features = false }
 num-bigint = "0.4"
 processor = { package = "miden-processor", path = "../processor", version = "0.8", features = ["internals"], default-features = false }
+miden-core = { package = "miden-core", path = "../core", version = "0.8", default-features = false }
```

Comment:
I think this is needed for `Smt` import? If so, I'd rather pull it through `test-utils` (that's how we do it for other similar structs).

Reply:
Right - fixed